### DISCUSSION TITLE
credential-file-method vs credential-process-method?

### DISCUSSION THREAD

**Discussion by jalaziz** [Noovolari/leapp#272] [Q&A] [answered]
What's the difference? I can't find any documentation on the difference and the commit that introduced it has a very vague commit message.

Is there a reason to use one over the other?

**Comment by urz9999** [comment, marked as answer]
Credential Process is another way to generate credentials for AWS related things, like CLI or SDK, the important thing is: NO LONG OR SHORT lived credentials are written in your machine. With credential process everytime CLI,SDK, CDK or any other tool (aware of this method) requests access to AWS credentials are generated on the fly and passed transparently. In a way is much more secure.
